.github/workflows/lint.yml

```diff
@@ -7,18 +7,24 @@ on:
     branches:
       - main
 
+permissions:
+  contents: read
+
 jobs:
   actionlint:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+        with:
+          persist-credentials: false
       - name: Enable action lint matcher
-        run:  echo "::add-matcher::.github/actionlint-matcher.json"
+        run: echo "::add-matcher::.github/actionlint-matcher.json"
       - name: Download actionlint
         id: get_actionlint
         # v1.7.4
         run: bash <(curl https://raw.githubusercontent.com/rhysd/actionlint/5db9d9cde2f3deb5035dea3e45f0a9fff2f29448/scripts/download-actionlint.bash)
         shell: bash
       - name: Check workflow files
+        # zizmor: ignore[template-injection]
```

Comment:
is this needed? if so it'd be worth a comment why

Reply:
I'll add a comment and will try to fix in a follow-up PR (this is the action from actionlint's setup guide)

Reply:
Added!